maven-release-manager/src/main/java/org/apache/maven/shared/release/phase/RewritePomsForDevelopmentPhase.java

```diff
@@ -65,7 +65,19 @@ protected void transformScm( MavenProject project, Model modelTarget, ReleaseDes
                         scmRoot.setConnection( scm.getConnection() );
                         scmRoot.setDeveloperConnection( scm.getDeveloperConnection() );
                         scmRoot.setUrl( scm.getUrl() );
-                        scmRoot.setTag( translator.resolveTag( scm.getTag() ) );
+                        String tag = translator.resolveTag( scm.getTag() );
```

Comment:
Still, can you take a look at `CvsScmTranslator`? It basically does what you do here. Can this be limited insite the Subversion translator?

Reply:
Why? This behavior makes sense for every ScmTranslator not supporting tag resolving. I would rather remove the handling from Cvs and return null there instead.

Reply:
@michael-o Ping. WDYT about my previous comment? Any chance to include this in the upcoming M6?

Reply:
No, not yet. I want to finish Fluido first and then go over to MRELEASE.